### ISSUE TITLE
Does anyone interested in a new modern version ?

### ISSUE THREAD

**Issue by EslaMx7** [EslaMx7/ScreenTask#24]
**Hello**,

Thank you all for using this application, it's been more than _**4 years**_   since the first version, I hope you enjoyed using it, I truly appreciate every single feedback from you.

I've learned a lot since then, and I was thinking if I could improve the idea much more.

Do you think that the idea of this application needs to be re-written in a new modern professional way?

I want to know if you are still interested in it because when I take a look at the project traffic I notice there are new users every day comes from search engines and various forums.

If you have any ideas that you wish to be in the new version, I will very pleased to know.

Looking forward your appreciated feedback 😊 

Best Regards,
Eslam
_Author of ScreenTask_

**Comment by Galenvan** [comment]
I would really like a new version of this application.  It would be great if there was an option to send the mouse position and clicks to the machine running ScreenTask for remote control.  I'd like for the option to use it over the Internet to be standard.  (I have seen that feature in the Issues section - Issue #24 to be exact - and was going to try to get someone to compile the necessary code for me.)  It would also be good if it was fully mobile compatible.

I run several machines, so I would be happy to be a tester.

**Comment by acesTech** [comment]
A new version would be awesome! I use your application every day in my computer lab and would love to see some new features.

**Comment by 15w34r0n3D4y** [comment]
Good tool, still works tested on Win7, thank you.

**Comment by i36** [comment]
A new version would be amazing!Addition possibility to select the screens (multiscreen)  and addition possibility to select area of the screen!AND add  language file in English so I can try to translate it into Chinese.I'm not a developer but a user,so my ability to read code is limited.BEST WISHS!

**Comment by masterxavierfox** [comment]
Yes please especially one with option to select multiple screens to cast to. Great software btw.

**Comment by KodexKy** [comment]
Yes please.

**Comment by kgnsflint96** [comment]
Yes, I would be interested. Specifically, if you could create a video wall that could show all of the desktops your client is viewing. You could then click on specific screens to view fullscreen as needed. This would be great for help desks to watch users in classroom settings.

**Comment by hdelei** [comment]
Yeah, you did a great job. I think we could use the images to turn into some kind of stream protocol to send to the recording servers as DVR. Thus, users can record screen activities for audit purposes.

**Comment by excellentworth** [comment]
This project has been very helpful, however as another issue reported what would make this ideal would be to add the following 3 features to on startup  Set default IP and port, start server at launch, run in background.  Ideally I would like to run this application as a service.

**Comment by samysousa** [comment]
Absolutely. A local realtime streaming server would be amazing.
Something that can stream to VLC or KODI.
Protocol wise, internal to network, anything goes even RTSP or RTP could be used.

**Comment by kahaal** [comment]
Yes it's a great application, but several improvement could make it greater.

I work for a small NGO
our project are focused on Education Improvement at Madagascar, using ICT (computers in schools)
there Internet is available but too much expensive for the schools, so computers are only on a local network.

teachers use screentask to broadcast powerpoint presentation to students computers and it's great !
but it could be greater if it was possible to broadcast video with a 25fps and audio too (for example : local copy of a YouTube capture about course)

I want to add that the install and use are really easy and it's a good thing, 
no problem for the teachers who are not really used to complex install/use

last comment, the resolution could be a problem, for example when the host is a laptop with a 16/10 and the clients are desktop 4/3. A parameter to define screencast resolution could be nice.

Congratulation for your application
I am sure that i'll really enjoy the next version

**Comment by gsix14** [comment]
Yes a new version would be great. I have an issue where the fullscreen view is cropped at the top.

**Comment by ebube4im** [comment]
Hey, I am a Newbie here. I guess a new version will be cool. I really would love to see the Multi screen already raised in the issue section. Looking forward to getting on with the project

**Comment by Alexv15310** [comment]
Yes, sure !

**Comment by balhaija** [comment]
yes, diffidently. I am new to this but I think you are great in solving this problem

**Comment by cr500** [comment]
> Yes a new version would be great. I have an issue where the fullscreen view is cropped at the top.

Check scale & Layout on windows 10. make sure 100%

**Comment by shuhratjan** [comment]
> _THANK YOU_ @EslaMx7 , The app is the best i have seen, and so easy to use, to share screen.

**Comment by Alexv15310** [comment]
Yes, of course! It’s very interesting
  
>Среда, 25 сентября 2019, 9:58 +03:00 от SHUH <notifications@github.com>:
> 
>THANK YOU @EslaMx7 , The app is the best i have seen, and so easy to use, to share screen.
—
You are receiving this because you commented.
Reply to this email directly,  view it on GitHub , or  mute the thread .  
 
--
Александр Васильев

**Comment by gacoll** [comment]
I would say that a big advantage of this software is that it is small, quick and easy to use. When people ask for too much, they are probably expecting you to give them a replacement for large scale, expensive software like Teamviewer. The advantage you currently have is great. However, I have a request if I may, which is that when it opens, if it can open to full screen instantly. Currently you have to click the full screen button every time you launch program. Even a commandline would be suitable, or just remember the last setting when you exit. 
Kind Regards
Grant

**Comment by EslaMx7** [comment]
📢 [v1.2](https://github.com/EslaMx7/ScreenTask/releases/tag/v1.2) Released! 🔥🔥🔥
